_posts/en/newsletters/2023-03-22-newsletter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 'Bitcoin Optech Newsletter #243'
+permalink: /en/newsletters/2023/03/22/
+name: 2023-03-22-newsletter
+slug: 2023-03-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+*No significant news this week was found on the Bitcoin-Dev or
+Lightning-Dev mailing lists.*
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Xapo Bank supports Lightning:**
+  Xapo Bank [announced][xapo lightning blog] its customers can now send outgoing
+  Lightning payments from the Xapo Bank mobile apps, using underlying infrastructure from Lightspark.
+
+- **TypeScript library for miniscript descriptors released:**
+  The TypeScript-based [Bitcoin Descriptors Library][github descriptors library]
+  has support for [PSBTs][topic psbt], [descriptors][topic descriptors],
+  [miniscript][topic miniscript], and signing support including some hardware
+  signing devices.
```

Comment:
This feels a bit unclear. The enumeration describes a few different technologies, but the signing support seems to refer to all of the former as well as the following hardware signing devices. Perhaps splitting the sentence would clarify:

```suggestion
  The TypeScript-based [Bitcoin Descriptors Library][github descriptors library]
  has support for [PSBTs][topic psbt], [descriptors][topic descriptors],
  and [miniscript][topic miniscript]. This includes support for signing directly, or via some hardware signing devices.
```